src/main/java/org/jenkinsci/test/acceptance/plugins/git/GitRepo.java

```diff
@@ -94,16 +94,22 @@ public void setIdentity(String userName, String userMail) {
     private File initDir() {
         try {
             // FIXME: perhaps this logic that makes it use a separate key should be moved elsewhere?
-            privateKey = File.createTempFile("ssh", "key");
-            FileUtils.copyURLToFile(GitContainer.class.getResource("GitContainer/unsafe"), privateKey);
-            Files.setPosixFilePermissions(privateKey.toPath(), singleton(OWNER_READ));
+            privateKey = Files.createTempFile("ssh", "key").toFile();
 
-            ssh = File.createTempFile("jenkins", "ssh");
-            FileUtils.writeStringToFile(ssh,
-                    "#!/bin/sh\n" +
-                            "exec ssh -o StrictHostKeyChecking=no -i " + privateKey.getAbsolutePath() + " \"$@\"" , StandardCharsets.UTF_8);
-            Files.setPosixFilePermissions(ssh.toPath(), new HashSet<>(Arrays.asList(OWNER_READ, OWNER_EXECUTE)));
+            FileUtils.copyURLToFile(GitContainer.class.getResource("GitContainer/unsafe"), privateKey);
 
+            if (SystemUtils.IS_OS_WINDOWS) {
+                ssh = Files.createTempFile("jenkins", "ssh.cmd").toFile();
```

Comment:
windows does not have an `executable` bit per say - it is based on the extension so ensure we end with `.cmd`